tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -27,7 +28,8 @@
 #include "TestUtils.h"
 
 TEST(NSFileManager, GetAttributes) {
-    NSString* testFileFullPath = getPathToFile(@"/data/NSFileManagerUT.txt");
+    NSString* testFileFullPath =
+        [[NSString stringWithUTF8String:GET_CURRENT_TEST_DIRECTORY().c_str()] stringByAppendingPathComponent:@"/data/NSFileManagerUT.txt"];
```

Comment:
I still like the idea of there being one nice shared _thing_ that does this.. but for "lowest common denominator" support it would have to be either CFString or std::string/std::wstring returning. The CG.Drawing tests run as C++ only.